src/components/DataTable/DataTable.jsx

```diff
@@ -0,0 +1,260 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import { findAllChildComponents } from '../../util/child-component';
+
+import Checkbox from '../Checkbox/Checkbox';
+import ScrollTable from '../ScrollTable/ScrollTable';
+
+const {
+	Thead,
+	Tbody,
+	Tr,
+	Th,
+	Td
+} = ScrollTable;
+
+const boundClassNames = lucidClassNames.bind('&-DataTable');
+
+const {
+	any,
+	func,
+	object,
+	string,
+	bool,
+	arrayOf,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["table"], "madeFrom": ["Table","ScrollTable"]}
```

Comment:
i can update to this, but i still think Table is useful here since they need to reference Table.Th to see what props are available to `Column` and `ColumnGroup`. Also Checkbox, while it's true it's make from it, doesn't expose any of it's props thru this abstraction, so it may not be very useful to the reader of the docs.